testsuite/MDAnalysisTests/lib/test_pkdtree.py

```diff
@@ -92,102 +93,100 @@ def test_set_coords():
 boxes_2 = ([10, 10, 10, 90, 90, 90],  # ortho
           [10, 10, 10, 45, 60, 90])  # tri_box
 
-# Find images for a given query vector, here in fractional coordinates
-queries = ([0.5, 0.5, 0.5],  # case box center
-           [0.1, 0.5, 0.5],  # box face
+# Find images for several query points, here in fractional coordinates.
+queries = ([0.1, 0.5, 0.5],  # box face
+           [0.5, 0.5, 0.5],  # box center
            [0.5, -0.1, 0.5],  # box face
            [0.1, 0.1, 0.5],  # box edge
            [0.5, -0.1, 1.1],  # box edge
            [0.1, 0.1, 0.1],  # box vertex
            [0.1, -0.1, 1.1],  # box vertex
            [2.1, -3.1, 0.1]  # box vertex
            )
-# Images for the previous query vectors, here in fractional coordinates
-centers = (([0.5, 0.5, 0.5], ),
-           ([0.1, 0.5, 0.5], [1.1, 0.5, 0.5]),
-           ([0.5, 0.9, 0.5], [0.5, -0.1, 0.5]),
-           ([0.1, 0.1, 0.5], [1.1, 0.1, 0.5],
-            [0.1, 1.1, 0.5], [1.1, 1.1, 0.5]),
-           ([0.5, 0.9, 0.1], [0.5, -0.1, 0.1],
-            [0.5, 0.9, 1.1], [0.5, -0.1, 1.1]),
-           ([0.1, 0.1, 0.1], [1.1, 0.1, 0.1], [0.1, 1.1, 0.1],
-            [0.1, 0.1, 1.1], [0.1, 1.1, 1.1], [1.1, 1.1, 0.1],
-            [1.1, 0.1, 1.1], [1.1, 1.1, 1.1]),
-           ([0.1, 0.9, 0.1], [1.1, 0.9, 0.1], [0.1, -0.1, 0.1],
-            [0.1, 0.9, 1.1], [0.1, -0.1, 1.1], [1.1, -0.1, 0.1],
-            [1.1, 0.9, 1.1], [1.1, -0.1, 1.1]),
-           ([0.1, 0.9, 0.1], [1.1, 0.9, 0.1], [0.1, -0.1, 0.1],
-            [0.1, 0.9, 1.1], [0.1, -0.1, 1.1], [1.1, -0.1, 0.1],
-            [1.1, 0.9, 1.1], [1.1, -0.1, 1.1])
-           )
-
 
-@pytest.mark.parametrize('b, qcs', product(boxes_2, zip(queries, centers)))
-def test_find_centers(b, qcs):
+# Images for the previous query vectors, here in fractional coordinates.
+images = (([1.1, 0.5, 0.5],),
+          (),
+          ([0.5, -0.1, 0.5],),
+          ([1.1, 0.1, 0.5], [0.1, 1.1, 0.5], [1.1, 1.1, 0.5]),
+          ([0.5, -0.1, 0.1], [0.5, 0.9, 1.1], [0.5, -0.1, 1.1]),
+          ([1.1, 0.1, 0.1], [0.1, 1.1, 0.1], [0.1, 0.1, 1.1],
+           [0.1, 1.1, 1.1], [1.1, 1.1, 0.1], [1.1, 0.1, 1.1],
+           [1.1, 1.1, 1.1]),
+          ([1.1, 0.9, 0.1], [0.1, -0.1, 0.1], [0.1, 0.9, 1.1],
+           [0.1, -0.1, 1.1], [1.1, -0.1, 0.1], [1.1, 0.9, 1.1],
+           [1.1, -0.1, 1.1]),
+          ([1.1, 0.9, 0.1], [0.1, -0.1, 0.1], [0.1, 0.9, 1.1],
+           [0.1, -0.1, 1.1], [1.1, -0.1, 0.1], [1.1, 0.9, 1.1],
+           [1.1, -0.1, 1.1]))
+
+
+@pytest.mark.parametrize('b, qcs', product(boxes_2, zip(queries, images)))
+def test_find_images(b, qcs):
     """
-    Test the generation of images for a given query vector and type of box
-    :param b: box as a list with six items
-    :param qcs: a query vector and a list of expected images
+    Test the generation of images for a given query vector and type of box.
+    :param b: box as a list with six items.
+    :param qcs: a query vector and a list of expected images.
```

Comment:
we should use numpy doc string convention everywhere. In a test a normal comment is enough though.  Also much better here is that `b` contains a mdanalysis dimensions variable. (x,y,z, alpha, beta, gamma)